packages/plugins/experimental/plugin-calls/src/state/call-manager.ts

```diff
@@ -0,0 +1,184 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+import { type PublicKey, type Client } from '@dxos/client';
+import { Resource } from '@dxos/context';
+import { create } from '@dxos/live-object';
+import { isNonNullable } from '@dxos/util';
+
+import { CallSwarmSynchronizer, type CallState } from './call-swarm-synchronizer';
+import { MediaManager, type MediaState } from './media-manager';
+import { type TranscriptionState, CALLS_URL, type EncodedTrackName, TrackNameCodec } from '../types';
+
+export type GlobalState = { call: CallState; media: MediaState };
+
+/**
+ * Top level manager for call state.
+ */
+export class CallManager extends Resource {
+  /**
+   * Live object state. Is changed on internal events.
+   * CAUTION: Do not change directly.
+   */
+  private readonly _state = create<GlobalState>({
+    call: {},
+    media: { pulledAudioTracks: {}, pulledVideoStreams: {} },
+  });
+
+  private readonly _swarmSynchronizer: CallSwarmSynchronizer;
+  private readonly _mediaManager: MediaManager;
+
+  get raisedHand() {
```

Comment:
we've been adding `/** @reactive */` jsdocs to methods/getters which access live objects